scheduler/docs/configuration.asc

```diff
@@ -27,6 +27,9 @@ The next several sections will document what every option does.
 `:port`::
   This is the port that the REST API will bind to.
 
+`:hostname`::
+  This is the hostname of the server Cook is running on. Defaults to "localhost"
```

Comment:
Need to correct the "Defaults to" since we changed the default

Reply:
done